composer/trainer/trainer.py

```diff
@@ -2396,7 +2402,12 @@ def save_checkpoint(self, name: str = 'ep{epoch}-ba{batch}-rank{rank}', *, weigh
         Returns:
             str or None: See :func:`.save_checkpoint`.
         """
-        return save_checkpoint(state=self.state, filename=name, weights_only=weights_only)
+        return save_checkpoint(
```

Comment:
any way we can fix both functions have the same name?

Reply:
I changed it to be:
```
def save_checkpoint(..):
    return checkpoint.save_checkpoint(...)
```

to clarify the save_checkpoint namespace a bit, does that work?